ui/store/actions.ts

```diff
@@ -4250,6 +4250,12 @@ export function setNewPrivacyPolicyToastClickedOrClosed() {
   };
 }
 
+export function setOnboardingDate() {
```

Comment:
is it possible to add a unit test for this function on the `action.test.js` ? should be sample but better for the test coverage